README.md

```diff
@@ -12,7 +12,20 @@
 $ sh ./gradlew -Pjdk18_home=<jdk18_home_dir> -Pllvm_home=<libclang_dir> clean verify
 ```
 
-After building, there should be a new `jextract` folder under `build` (the contents and the name of this folder might vary slightly depending on the platform):
+<details><summary>Using a local installation of LLVM</summary>
+
+While the recommanded way is to use a [release from the LLVM project](https://releases.llvm.org/download.html),
+extract it then make `llvm_home` point to this directory, it may be possible to use a local installation instead.
+
+E.g. on macOs the `llvm_home` can also be set as one of these locations :
+
+* `/Library/Developer/CommandLineTools/usr/` if using Command Line Tools
+* `/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/` if using XCode
+* `$(brew --prefix llvm)` if using the [LLVM install from Homebrew](https://formulae.brew.sh/formula/llvm#default)
+
+</details>
+
+After building, there should be a new `jextract` folder under `build` (the contents and the name of this folder might vary slightly depending on the platform, e.g. on macOs the folder is `jextract.app`) :
```

Comment:
Do you think it is still necessary to provide specific details about macOs here (and below) ?

Reply:
Yes I think this is necessary as long as this is packaged with `jpackage`, because not everyone is accustomed ta macOs application layout.

In my opinion jpackage does a very neat job to pack desktop application. But for command line interactions this is less practical in my opinion. I think this might complicate a bit the consumption of this command line tool, requiring to adapt the actual path, that unless a workaround is set up (a symbolic link, or adding `.../jextract.app/Contents/MacOS` to `PATH`).


Ultimately I think a `.tar.gz` distribution would be better for consumption, but I don't know to do that (and elegantly) at this time.

Reply:
IMHO, the location of the jextract packaged application is not too important. In general I would expect users to build jextract once, and then install it in some well known path, at which point the name of the folder chosen by jpackage becomes irrelevant (which is why I'm not sure it's worth spending extra verbiage for this). What we might want to do is to add a post jpackage step which renames the folder to something consistent across platforms.

Reply:
> What we might want to do is to add a post jpackage step which renames the folder to something consistent across platforms.

At this time I don't think it's possible to do that with the output of jpackage, on macOs `jextract` binary will lookup for its config file at `../../Contents/app/jextract.cfg`, then `../../Contents/runtime`, etc.


```shell
$ tree build/jextract.app
build/jextract.app
└── Contents
    ├── Info.plist
    ├── MacOS
    │   └── jextract
    ├── PkgInfo
    ├── Resources
    │   └── jextract.icns
    ├── app
    │   └── jextract.cfg
    └── runtime
        └── Contents
            ├── Home
            │   ├── bin
            │   │   ├── java
            │   │   ├── javac
            │   │   ├── keytool
            │   │   └── serialver
            │   ├── conf
            │   │   ├── jextract
            │   │   │   ├── __clang_cuda_builtin_vars.h
            │   │   │   ├── __clang_cuda_cmath.h
            │   │   │   └── ...
            │   │   ├── net.properties
            │   │   └── security
            │   │       ├── java.policy
            │   │       ├── java.security
            │   │       └── policy
            │   │           └── ...
            │   ├── include
            │   │   ├── classfile_constants.h
            │   │   ├── darwin
            │   │   │   └── jni_md.h
            │   │   ├── jni.h
            │   │   ├── jvmti.h
            │   │   └── jvmticmlr.h
            │   ├── legal
            │   │   ├── ...
            │   │   └── jdk.incubator.foreign
            │   │       ├── ADDITIONAL_LICENSE_INFO
            │   │       ├── ASSEMBLY_EXCEPTION
            │   │       └── LICENSE
            │   ├── lib
            │   │   ├── classlist
            │   │   ├── ct.sym
            │   │   ├── jrt-fs.jar
            │   │   ├── jspawnhelper
            │   │   ├── jvm.cfg
            │   │   ├── libLLVM.dylib
            │   │   ├── libclang.dylib
            │   │   ├── libjava.dylib
            │   │   ├── libjimage.dylib
            │   │   ├── libjli.dylib
            │   │   ├── libjsig.dylib
            │   │   ├── libnet.dylib
            │   │   ├── libnio.dylib
            │   │   ├── libosxsecurity.dylib
            │   │   ├── libprefs.dylib
            │   │   ├── libsyslookup.dylib
            │   │   ├── libverify.dylib
            │   │   ├── libzip.dylib
            │   │   ├── modules
            │   │   ├── security
            │   │   │   ├── blocked.certs
            │   │   │   ├── cacerts
            │   │   │   ├── default.policy
            │   │   │   └── public_suffix_list.dat
            │   │   ├── server
            │   │   │   ├── libjsig.dylib
            │   │   │   └── libjvm.dylib
            │   │   └── tzdb.dat
            │   └── release
            ├── Info.plist
            └── MacOS
                └── libjli.dylib

27 directories, 233 files
```